packages/forms/src/index.js

```diff
@@ -5,25 +5,21 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import * as styledElements from './styled';
-
 /** Common */
-export { default as Field } from './fields/common/Field';
-export { default as Hint } from './fields/common/Hint';
-export { default as Label } from './fields/common/Label';
-export { default as Message } from './fields/common/Message';
-
-/** Other */
-export { default as FauxInput } from './fields/other/FauxInput';
-export { default as MediaInput } from './fields/other/MediaInput';
+export { default as Field } from './elements/common/Field';
+export { default as Hint } from './elements/common/Hint';
+export { default as Label } from './elements/common/Label';
+export { default as Message } from './elements/common/Message';
 
 /** Fields */
-export { default as Checkbox } from './fields/Checkbox';
-export { default as Input } from './fields/Input';
-export { default as Radio } from './fields/Radio';
-export { default as Textarea } from './fields/Textarea';
-export { default as Toggle } from './fields/Toggle';
-export { default as Range } from './fields/Range';
-export { default as MultiThumbRange } from './fields/MultiThumbRange';
+export { default as Checkbox } from './elements/Checkbox';
+export { default as Input } from './elements/Input';
+export { default as Radio } from './elements/Radio';
+export { default as Range } from './elements/Range';
+export { default as Textarea } from './elements/Textarea';
+export { default as Toggle } from './elements/Toggle';
 
-export const styled = styledElements;
+/** Other */
+export { default as FauxInput } from './elements/FauxInput';
+export { default as MediaInput } from './elements/MediaInput';
+export { default as MultiThumbRange } from './elements/MultiThumbRange';
```

Comment:
Even though this isn't backed by a native `<input />` I still classify the `MultiThumbRange` as a field. Should this be moved up to the `Fields` block?